ansible_ai_connect/ai/api/views.py

```diff
@@ -162,11 +208,14 @@ def post(self, request) -> Response:
         return pipeline.execute()
 
 
-class Feedback(APIView):
+class Feedback(OurAPIView):
     """
     Feedback API for the AI service
     """
 
+    serializer_class = FeedbackRequestSerializer
```

Comment:
This view is still handling de-serialization and validation [itself](https://github.com/ansible/ansible-ai-connect-service/pull/1147/files#diff-ecfb6919dfd8379aafba96af7457b253e4dce528897dfe6bfc207ca2b3b2ada9R242-R247) (in addition to that in `OurAPIView`).

Reply:
Done, thanks.